src/open_company/api/companies.clj

```diff
@@ -96,26 +100,75 @@
   :delete! (fn [_] (company/delete-company slug))
 
   ;; Create or update a company
+  ;; TODO remove possibility to create company
```

Comment:
So... in theory PUT would be an alternative way for them to specify the desired slug and is a reasonable way to create the company with the API. If we allow it, we need to pull the desired slug from the URL, as if they provided it in the POSTed body, and we'd need to look for a self-conflicting request where they provide a slug by PUTing and in the body of the PUT request, but they don't match.